cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/sinks/NGSIDynamoDBSink.java

```diff
@@ -49,7 +49,7 @@
      * Available DynamoDB regions implementation.
```

Comment:
Could you add a CHANGE_NEXT_RELEASE line bout the fix, e.g:

```
[cygnus-ngsi] [NGSIDynamoDBSink] [bug] Southeast regions are bad coded (#1448)
```

Once this gets done, PR will be merged.

Reply:
Fixed in 2ba941b